### PR TITLE
Useless cat, consider using grep -c instead of grep|wc, adding double quote to prevent globbing and word splitting, and update PHP version 7.2.2

### PR DESCRIPTION
**Pull request by sptndc**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 cd "$(dirname "$0")"
 
 # Use all cores for the build process
-CORE_COUNT=$(cat /proc/cpuinfo | grep processor | wc -l)
```
Comment (kasparsd) [inline build.sh:5]: @sptndc I'm failing to see any output when running `< /proc/cpuinfo | grep -c processor -l`. Same with `echo $(< /proc/cpuinfo | grep -c processor -l)`.

Secondly, `-l` in grep is for "Display the filenames, but do not display the matched lines." -- is that the intended use-case or a left-over from the `wc -l`?

**Review by kasparsd — COMMENTED** [review]
Thanks for the pull request @sptndc!

I had only one question posted inline the changeset.

File: build.sh (hunk 1 above)
Comment (sptndc) [reply build.sh:5]: Sorry I forgot to remove `-l` from there.

Comment (kasparsd) [reply build.sh:5]: `echo $(< /proc/cpuinfo | grep -c processor)` still returns `0` on a Debian Wheezy box so I'll revert it back to `cat` for now.

Comment (sptndc) [reply build.sh:5]: Ok, no problem. It's just an issue report from `shellcheck`.

Comment (kasparsd) [reply build.sh:5]: That's a great idea -- we should probably make Travis run the `spellcheck` on pull requests.

Comment (sptndc) [reply build.sh:5]: Yes, that's better.

**Pull request merged**